api/api.go

```diff
@@ -259,17 +258,18 @@ func (a *API) getConfig(ctx context.Context) *conf.Configuration {
 
 	config := obj.(*conf.Configuration)
 
-	extConfig := (*a.config).External
-	if err := mergo.MergeWithOverwrite(&extConfig, config.External); err != nil {
-		return nil
-	}
-	config.External = extConfig
+	// Merge global & per-instance external config for multi-instance mode
+	// extConfig := (*a.config).External
+	// if err := mergo.MergeWithOverwrite(&extConfig, config.External); err != nil {
+	// 	return nil
+	// }
+	// config.External = extConfig
 
-	smtpConfig := (*a.config).SMTP
-	if err := mergo.MergeWithOverwrite(&smtpConfig, config.SMTP); err != nil {
-		return nil
-	}
-	config.SMTP = smtpConfig
+	// smtpConfig := (*a.config).SMTP
+	// if err := mergo.MergeWithOverwrite(&smtpConfig, config.SMTP); err != nil {
+	// 	return nil
+	// }
+	// config.SMTP = smtpConfig
```

Comment:
Could there be a comment on why this is commented out instead of removed entirely?

Reply:
oh there's a comment on line 261 to explain what it does, but i'll make it more explicit to explain why it's commented out

Reply:
realised i don't have to comment it out in the end, just needed to check if it's in multi-instance mode before deciding whether to merge the respective configs